tests/Persisters/FlysystemTest.php

```diff
@@ -0,0 +1,187 @@
+<?php
+
+namespace Rubix\ML\Tests\Persisters;
+
+use Rubix\ML\Persistable;
+use Rubix\ML\Persisters\Flysystem;
+use Rubix\ML\Persisters\Persister;
+use Rubix\ML\Classifiers\DummyClassifier;
+use League\Flysystem\Filesystem;
+use League\Flysystem\FilesystemOperator;
+use League\Flysystem\InMemory\InMemoryFilesystemAdapter;
+use League\Flysystem\UnableToWriteFile;
+use League\Flysystem\UnableToMoveFile;
+use PHPUnit\Framework\TestCase;
+use RuntimeException;
+
+/**
+ * @group Persisters
+ * @covers \Rubix\ML\Persisters\Flysystem
+ */
+class FlysystemTest extends TestCase
+{
+    /**
+     * The path to the test file.
+     *
+     * @var string
+     */
+    const PATH = __DIR__ . '/path/to/test.persistable';
```

Comment:
Did we want to use the temp directory here instead?

Reply:
I think as we're using the `InMemoryFilesystemAdapter` for this test we needn't worry about providing a writable location on the local filesystem.

When we prep a PR to include this feature in the main repo we should also include the changes to the `Filesystem` persister to extend `Flysystem`. I think it would be good then to have that unit test substitute `__DIR__` for `__TMP__` (a new constant that we'll define in global namespace in `src/constants.php`)